sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetAvroCompatibilitySuite.scala

```diff
@@ -62,12 +70,12 @@ class ParquetAvroCompatibilitySuite extends ParquetCompatibilityTest with Shared
               .setDoubleColumn(i.toDouble + 0.2d)
               .setBinaryColumn(ByteBuffer.wrap(s"val_$i".getBytes(StandardCharsets.UTF_8)))
               .setStringColumn(s"val_$i")
+              .setFixedColumn(new FixedType(generateFixedLengthByteArray(i)))
               .build())
         }
       }
 
       logParquetSchema(path)
-
       checkAnswer(spark.read.parquet(path), (0 until 10).map { i =>
```

Comment:
I think this only tests against `spark.sql.parquet.enableVectorizedReader` is true (default), should we test non-vectorized reader too?